.github/workflows/integration-test-containers.yml

```diff
@@ -45,6 +45,7 @@ jobs:
               - integration-tests/Dockerfile
               - .github/workflows/integration-test-containers.yml
 
+  # Trigger rebuild
```

Comment:
Shouldn't it be rebuilding anyway, or is the list of filters above not working properly?

Reply:
It is not triggering because this PR doesn't have any changes to the files in the filter. I was gonna suggest we should add the `base` argument, but the docs say this is ignored when the trigger is `pull_request`, so not entirely sure what we could do about it 🤷🏻‍♂️